### PR TITLE
add cbac documentation

### LINKED ISSUE #439: ABAC
ABAC documentation in general.
**Wait until ABAC is done in the GUI!**

### PR DESCRIPTION
**Pull request by david-fritz-gravwell**
Fixes #439

### CODE REVIEW

**Review by ashnwade — CHANGES_REQUESTED** [review]
A few minor changes

File: cbac/cbac.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Users that don't have access to a particular feature will see a menu system with those features disabled. For example, a user that does not have access to dashboard or data ingest will see a menu system like this:
+
+![CBAC Menu](cbac_menu.png)
+
+## Granting Capabilities in Practice
+
+A typical use of user and group CBAC grants is to not provide any grants to individual users, and instead create groups with specific roles. For example, creating a group named "IT Users" that has access to IT related data (syslog, router logs, firewall logs, etc.), and a group named "Incident Response Users" that has access to IDS and other security related data, allows the admin to grant access to users based on their role. Users that need access to both IT and Incident response data in this example can simply be added to both groups.
+
+## List of CBAC Capabilities
```
[the remark this answers is not in the document]
Comment (david-fritz-gravwell) [reply cbac/cbac.md:35]: I copy pasted the list from the typescript in the frontend PR that updated the language.

**Comment by david-fritz-gravwell** [comment]
@john-floren-gravwell @ashnwade PTAL

**Review by john-floren-gravwell — APPROVED** [review]

**Review by ashnwade — APPROVED** [review]
LGTM

**Pull request merged**